### PR TITLE
CSS Linting.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Blocks and component CSS linting.

### Types of changes

- Improvement

### How has this been tested?

- Drag and drop the respective block and check everything works correctly or not.

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: dist/common-editor.css [hunk 1]
```diff
@@ -119,25 +119,19 @@
   color: #999;
   display: inline-block;
   height: 16px;
-  width: 22px;
```
Comment (RajkiranBagal) [inline dist/common-editor.css:122]: We will need the below CSS rather than the above CSS.

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/spacing-control/editor.lazy.scss [hunk 2]
```diff
@@ -37,23 +41,11 @@
 		width: 25%;
 	}
 
-	.uagb-spacing-control__link {
-		width: 12%;
-		background: #d5dadf;
```
Comment (RajkiranBagal) [inline src/components/spacing-control/editor.lazy.scss:42]: WHat about the bg css?

**Review by RajkiranBagal — CHANGES_REQUESTED** [review]

File: src/components/spacing-control/editor.lazy.scss (hunk 2 above)
Comment (AbhijitNage123) [reply src/components/spacing-control/editor.lazy.scss:42]: Updated

File: dist/common-editor.css (hunk 1 above)
Comment (AbhijitNage123) [reply dist/common-editor.css]: Updated

**Review by sandeshjangam — APPROVED** [review]

**Pull request merged**